bot/code_review_bot/__init__.py

```diff
@@ -88,7 +88,7 @@ def __init__(
 
         # Support line 0, with a Sentry warning
```

Comment:
Can you adjust the comment to explain that we get line 0 for full-file issues? And remove the part "with a Sentry warning"